dev.yml

```diff
@@ -7,7 +7,7 @@ up:
     - openssl
     - sqlite
   - ruby:
-      version: 2.2.3p172-shopify
+      version: 2.2.3
```

Comment:
2.2.x wasnt allowed before because Shopify used a super special snowflake 2.2.x that was binary incompatible with standard 2.2.x